src/detect_breakends.cpp

```diff
@@ -23,13 +37,13 @@ auto enumeration_names(clustering_methods)
                                                   {"3", clustering_methods::candidate_selection_based_on_voting},
                                                   {"candidate_selection_based_on_voting",
                                                    clustering_methods::candidate_selection_based_on_voting}};
-}
+};
 
 struct cmd_arguments
 {
     std::filesystem::path alignment_file_path{};
     std::filesystem::path insertion_file_path{};
-    std::vector<uint8_t> methods{1, 2, 3, 4};                   // default is using all methods
+    std::vector<detecting_methods> methods{};
```

Comment:
```suggestion
    std::vector<detecting_methods> methods{cigar_string, split_read, read_pairs, read_depth};  // default is using all methods
```

Or does that look funny in the help page of the argument parser?

Reply:
No, I chose this way because otherwise we can never choose only one or two methods (see issue description). Before, the set methods were added to the default vector and not replaced the vector. If you know a better way, I'm all ears.

Reply:
Ah, that's very unintuitive and should be fixed if possible. https://github.com/seqan/seqan3/issues/2393

Reply:
This should work once https://github.com/seqan/seqan3/pull/2394 is merged

Reply:
@eseiler No its not 😞 
If you want to try, I added a debug_stream to check and you could test with:
```bash
lydia@alnus:~/build/iGenVar$ ./bin/detect_breakends ~/Repos/iGenVar/test/data/simulated.minimap2.hg19.coordsorted_cutoff.sam ~/Repos/iGenVar/test/data/out.fasta -m 1 -m 2
Methods to be used: [cigar_string,split_read,read_pairs,read_depth,split_read,read_pairs]
....
```